### PR TITLE
Fix NU1605 - ensure official builds override package versions correctly when building subfolders

### PR DESCRIPTION
**Pull request by natemcmaster**
The IISIntegration and WebSockets folders were not getting the right versions when building in official builds. This was caused by importing a dependencies.props file which reset correct version variables to invalid ones.

### CODE REVIEW

File: build/dependencies.folderbuilds.props [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿<!-- This file is temporary while we rework dependencies management. It is used when building individual folders as if they were repos. -->
```
Comment (jkotalik) [inline build/dependencies.folderbuilds.props]: How long will this need to exist? What about other folders (like DataProtection)?

Comment (jkotalik) [reply build/dependencies.folderbuilds.props]: Nvrm I see data protection was also updated.

File: build/RepositoryBuild.targets [hunk 2]
```diff
@@ -78,6 +78,7 @@
       <RepositoryBuildArguments Condition="'$(CI)'== 'true'">$(RepositoryBuildArguments) -DotNetHome '$(DOTNET_HOME)'</RepositoryBuildArguments>
       <!-- Should reduce allowable package feeds to only nuget.org. -->
       <RepositoryBuildArguments>$(RepositoryBuildArguments) /p:AspNetUniverseBuildOffline=true</RepositoryBuildArguments>
+      <RepositoryBuildArguments>$(RepositoryBuildArguments) /p:IsUniverseBuild=true</RepositoryBuildArguments>
```
Comment (jkotalik) [inline build/RepositoryBuild.targets]: Why is this needed?

Comment (natemcmaster) [reply build/RepositoryBuild.targets]: Ensures we don't import dependencies.folderbuilds.props during a "real build".

File: build/dependencies.folderbuilds.props (hunk 1 above)
Comment (natemcmaster) [reply build/dependencies.folderbuilds.props]: Hopefully not long. I'm going to work over the next few days to cleanup dependency and reference management.

File: src/IISIntegration/dependencies.overrides.props [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  <!-- These versions are pinned for backwards compatibility tests. -->
+  <PropertyGroup Label="Package Versions: Pinned">
+    <MicrosoftAspNetCoreAspNetCoreModuleStablePackageVersion>2.2.0-rtm-35665</MicrosoftAspNetCoreAspNetCoreModuleStablePackageVersion>
+    <MicrosoftAspNetCoreAspNetCoreModuleV2StablePackageVersion>2.2.0-rtm-35665</MicrosoftAspNetCoreAspNetCoreModuleV2StablePackageVersion>
+    <MicrosoftAspNetCoreServerIISStablePackageVersion>2.2.0-rtm-35665</MicrosoftAspNetCoreServerIISStablePackageVersion>
```
[the remark this answers is not in the document]
Comment (jkotalik) [reply src/IISIntegration/dependencies.overrides.props]: They are for regression tests to guarantee forward and back compatibility. They have Stable in their name and are Pinned.

**Comment by natemcmaster** [comment]
Merging to unblock CI for tonight's ProdCon build.
